include/xgboost/generic_parameters.h

```diff
@@ -76,18 +69,7 @@ struct GenericParameter : public XGBoostParameter<GenericParameter> {
     DMLC_DECLARE_FIELD(validate_parameters)
         .set_default(false)
         .describe("Enable checking whether parameters are used or not.");
-    DMLC_DECLARE_FIELD(n_gpus)
```

Comment:
Just to double-check, the generic parameter isn't part of the binary serialization, right?

Reply:
Binary model, no.   Binary pickle/saveRDS, yes.

Reply:
Well, at this point, we've given users sufficient time to migrate from saveRDS.